textarea/textarea_test.go

```diff
@@ -424,6 +329,8 @@ func TestView(t *testing.T) {
 		name      string
 		modelFunc func(Model) Model
 		expected  string
```

Comment:
If it's not too much work I would change expected to be:

```go
struct {
    value string
    cursorRow int
    cursorCol int
}
```

Reply:
Will do, makes much more  sense.

Any other changes you'd like?

Reply:
That's all, thank you for this PR!

Reply:
Decided to go with a common convention of putting everything in the struct `want`. I also decided to go `view` instead of `value` as that is actually what is stored in that string.